src/main/scala/is/hail/distributedmatrix/BlockMatrixIsDistributedMatrix.scala

```diff
@@ -0,0 +1,177 @@
+package is.hail.distributedmatrix
+
+import is.hail.utils._
+import org.apache.spark.rdd.RDD
+import org.apache.spark.mllib.linalg._
+import org.apache.spark.mllib.linalg.distributed._
+
+import scala.reflect.ClassTag
+
+object BlockMatrixIsDistributedMatrix extends DistributedMatrix[BlockMatrix] {
+  type M = BlockMatrix
+
+  def cache(m: M): M = m.cache()
+
+  def from(rdd: RDD[Array[Double]]): M =
+    new IndexedRowMatrix(rdd.zipWithIndex().map { case (x, i) => new IndexedRow(i, new DenseVector(x)) })
+      .toBlockMatrixDense()
```

Comment:
I figure a non-dense matrix should be created with some other method like:
```
def from(rdd: RDD[SparseVector]): M = ???
```
I just didn't implement that ;P